src/pybind/mgr/rbd_support/schedule.py

```diff
@@ -336,17 +293,9 @@ def next_run(self, now: datetime.datetime) -> str:
             raise ValueError('no items is added')
         return datetime.datetime.strftime(schedule_time, "%Y-%m-%d %H:%M:00")
 
-    def to_list(self) -> List[Dict[str, Optional[str]]]:
-        def item_to_dict(interval: Interval,
-                         start_time: Optional[StartTime]) -> Dict[str, Optional[str]]:
-            if start_time:
-                schedule_start_time: Optional[str] = start_time.to_string()
-            else:
-                schedule_start_time = None
-            return {SCHEDULE_INTERVAL: interval.to_string(),
-                    SCHEDULE_START_TIME: schedule_start_time}
-        return [item_to_dict(interval, start_time)
-                for interval, start_time in self.items]
+    def to_list(self) -> List[Dict[str, str]]:
+        return [{SCHEDULE_INTERVAL: interval.to_string()}
+                for interval in self.intervals]
```

Comment:
Do we still need to produce a list of dictionaries where the dictionary is basically redundant because it's guaranteed to have a single well-known key?  If the answer is yes, I'd stay with `items` and not rename to `intervals` because that would describe the structure better.